applications/serial_lte_modem/src/lwm2m_carrier/Kconfig

```diff
@@ -25,4 +25,14 @@ config SLM_CARRIER_AUTO_STARTUP
 	  Enable automatic startup of the library on device boot. If this configuration is
 	  disabled, automatic startup is controlled through a dedicated AT command.
 
+config SLM_CARRIER_FOTA_AUTOMATION
+	bool "Enable automatic handling after FOTA download"
+	depends on SLM_AUTO_CONNECT
+	default y
+	help
+	  Enable automatic reboot when requested by LwM2M stack. If this configuration is
+	  disabled, reboot is controlled by host MCU.
+	  Enable automatic link down/up when requested by LwM2M stack. If this configuration is
```

Comment:
`during a FOTA`?

Reply:
Automation for FOTA context only

Reply:
Yeah, that's why I think it would be good to make this more clear in the Kconfig option help message.